static/templates/settings_tab.handlebars

```diff
@@ -114,6 +114,15 @@
             {{t "Display time in 24-hour format (17:00 instead of 5:00 PM)" }}
           </label>
         </div>
+
+        <div class="control-group">
+          <label for="default_language" class="control-label">{{t "Default Language" }}</label>
+          <select name="default_language" id="default_language" value="{{ page_params.default_language }}">
+            <option selected="selected" value="en"> {{t "English" }}</option>
+            <option value="zh_CN"> {{t "Simplified Chinese" }}</option>
+          </select>
+        </div>
+
```

Comment:
We should add a little advertisement for Tranlating Zulip (maybe linked to http://zulip.readthedocs.io/en/latest/translating.html) on this section of the page.